_posts/en/newsletters/2018-10-30-newsletter.md

```diff
@@ -205,6 +205,7 @@ it to write a good description, and I doubt non-LN devs care -->{% endcomment %}
 - [LND #1981][] ensures that LND doesn't leak information about any of
   its peers that aren't advertising themselves as public nodes.
 
+{:#lnd-1535-1512}
```

Comment:
This splits the next bullet into its own list. Would it be better to just make `[LND #1535]` the id for the bullet?